### PR TITLE
Add Sentry on settings

### PR DESCRIPTION
**Pull request by kosssi**
This feature allow to activate or deactivate Sentry:

- [x] Add Sentry on onboarding
- [x] Add Sentry on settings
- [x] Add translations
- [x] Activate or deactivate Sentry according to choice
- [x] Update/Add tests

### CODE REVIEW

File: mobile/src/main.jsx [hunk 1]
```diff
@@ -23,14 +23,16 @@ import { initClient, initBar, isClientRegistered, resetClient, refreshFolder, on
...
+
 const loggerMiddleware = createLogger()
 
 const renderAppWithPersistedState = persistedState => {
   const store = createStore(
     filesApp,
     persistedState,
     applyMiddleware(
-      RavenMiddleWare(`https://${__SENTRY_TOKEN__}@sentry.cozycloud.cc/2`),
+      RavenMiddleWare(ANALYTIC_URL, getAnalyticConfiguration()),
```
Comment (enguerran) [inline mobile/src/main.jsx]: As we use this exact same parameter in `logException` in `crash_reporter.js`, we may refactor this. Check this for inspiration https://github.com/ngokevin/redux-raven-middleware/blob/master/index.es6

File: mobile/test/reducers/settings.spec.js [hunk 2]
```diff
@@ -14,6 +14,11 @@ describe('settings reducers', () => {
     .toEqual({serverUrl: url, error: null, authorized: false})
   })
 
+  it('should handle SET_ANALYTIC', () => {
+    expect(reducer({analytic: false}, {type: SET_ANALYTIC, analytic: true})).toEqual({analytic: true})
+    expect(reducer({analytic: true}, {type: SET_ANALYTIC, analytic: false})).toEqual({analytic: false})
+  })
```
Comment (enguerran) [inline mobile/test/reducers/settings.spec.js]: We'll refactor the whole `reducers` spec to test all-in-one: actions, action creators, reducers (cf. https://github.com/cozy/cozy-files-v3/commit/04c18b6b9522080fef6de0454fee03d7dca57bfa)

File: package.json [hunk 3]
```diff
@@ -139,6 +139,7 @@
     "globals": {
       "__ALLOW_HTTP__": false,
       "__TARGET__": "browser",
+      "__SENTRY_TOKEN__": "token",
```
Comment (enguerran) [inline package.json]: Why?

File: src/locales/en.json [hunk 4]
```diff
@@ -101,9 +101,15 @@
       },
       "photos": {
         "title": "Backup your photos and videos",
-        "description": "Automatically backup the photos taken with your phone to your Cozy, so you never lose them. By default, it will only be done when the WIFI is on. This setting can be changed later on.",
+        "description": "Automatically backup the photos taken with your phone to your Cozy, so you never lose them. By default, it will only be done when the WIFI is on. This setting can be changed later on."
```
Comment (enguerran) [inline src/locales/en.json]: Why do you add a coma in the end of this line?

Comment (kosssi) [inline src/locales/en.json]: I remove coma because buttons are now in "step" section ;)

File: package.json (hunk 3 above)
Comment (kosssi) [inline package.json]: It is the global variables of Jest. If I do not add it, the tests fail.

File: mobile/src/containers/onboarding/Analytic.jsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    dispatch(setAnalytic(true))
+    ownProps.nextStep()
+  },
+  onSkip: () => {
+    dispatch(setAnalytic(false))
+    ownProps.nextStep()
+  }
+})
+
+export default connect(mapStateToProps, mapDispatchToProps)(translate()(Analytic))
```
Comment (kosssi) [inline mobile/src/containers/onboarding/Analytic.jsx]: I copied the code from BackupPhotosVideos ;)

File: src/locales/en.json (hunk 4 above)
Comment (enguerran) [inline src/locales/en.json]: o_O I saw you added a coma, but in fact, you removed the coma...

**Comment by kosssi** [comment]
🎉

**Pull request merged**